api/src/pcapi/algolia_settings_collective_offers.json

```
@@ -25,7 +25,9 @@
         "filterOnly(offer.students)",
         "filterOnly(offer.subcategoryId)",
         "filterOnly(venue.departmentCode)",
-        "filterOnly(venue.id)"
+        "filterOnly(venue.id)",
+        "filterOnly(offer.educationalInstitutionUAICode)",
+        "filterOnly(offer.domains)",
```

Cette virgule finale, ça n'est pas du JSON valide, il me semble, non ?

```suggestion
        "filterOnly(offer.domains)"
```